src/compo/viirs_aod2ioda.py

```diff
@@ -13,10 +13,57 @@
 import numpy as np
 import os
 
+import glob
 import pyiodaconv.ioda_conv_engines as iconv
 from collections import defaultdict, OrderedDict
 from pyiodaconv.orddicts import DefaultOrderedDict
 
+
+def flag_by_confidence_lvl(qfarr, aerosol_type, conf_lvl='MedHigh', print_diag=False):
+
+    # ADP smoke confidence is defined in (counting from 0) bit 2-3, dust in bit 4-5
+    # input array contains decimal representations of the binary values
+    # 10 (2 in base10) is medium confidence and 00 (0 in base10) is high confidence
+    # 01 (1 in base10) is low confidence and 11 (3 in base10) is missing/bad data
+
+    if conf_lvl == 'med' or conf_lvl == 'medhigh':
+        conf_lvl = 'MedHigh'
+    if conf_lvl == 'low':
+        conf_lvl = 'Low'
+    if conf_lvl == 'high':
+        conf_lvl = 'High'
```

Comment:
```suggestion
    else:
        conf_lvl = conf_lvl.title()
```
This should force a capital letter and reduces the if statements